packages/ffe-tables/less/table.less

```diff
@@ -124,6 +124,11 @@
 
     &__caption {
         &:extend(.ffe-h4);
+        .native & {
```

Comment:
```
    .native & {
        @media (prefers-color-scheme: dark) {
            color: @ffe-white-darkmode;
        }
    }
```
Slik ser det ut i `typography.less`. Lite rart den blir mørkblå istellet for vit. Vad er det som overskriver dette? Jag er ikke noen less expert men tenkte dette burde blitt med når extend blir brukt.

Reply:
Ja, det er burde jo funke. Ser det samme er gjort i `components.less` for å override i darkmode.

Reply:
Du kan kanske se hos oss som har problemet vad det er som skriver over @ffe-white-darkmode hvis den gjør det.

Reply:
Slik er generert kode. Det ser alltså ikke ut som den darkmode blir extended. 
```
.emphasized-value__header,
.emphasized-value__value,
.ffe-calendar__title,
.ffe-card-component--title,
.ffe-context-message-content__header,
.ffe-h1,
.ffe-h2,
.ffe-h3,
.ffe-h4,
.ffe-h5,
.ffe-h6,
.ffe-table__caption,
.innholdsmelding__heading {
    color:#002776;
    font-family:MuseoSansRounded-700,arial,sans-serif;
    font-weight:400;
    margin-bottom:10px;
    margin-top:0
}
@media (prefers-color-scheme:dark) {
    .native .ffe-h1,
    .native .ffe-h2,
    .native .ffe-h3,
    .native .ffe-h4,
    .native .ffe-h5,
    .native .ffe-h6 {
        color:#fff
    }
}
```

Reply:
hmmm tror jag er inne på noe

` &:extend(.ffe-h4);` dette vil vell bare kopiera `.ffe-h4`. For og få darkmode trengs native. 

Kanske prova legge til 
```
 &:extend(.ffe-h4);
 &:extend(.native .ffe-h4);
```
Bare prøve.. googlet litt.